File: AR-VR/devilking0_alokprajapati_2024it1138_2/GettingStarted/gradingSystem.c
#include <stdio.h>



int main()
{
	int i;

	scanf("%d", &i);
	if (i > 90)
	{
		printf("excellent");
	}
	else if (i <= 90 && i > 80)
	{
		printf("good");
	}
	else if (i > 70 && i <= 80)
	{
		printf("fair");
	}
	else if (i > 60 && i <= 70)
	{
		printf("meets expectations");
	}
	else if (i <= 60)
	{
		printf("below par");
	}

	return 0;
}

File: IOT/Alphanupam_ANUPAMSHARMA_2024IT1165_2/GettingStarted/gradingsystem.java
import java.util.*;
  
  public class Main{
  
  public static void main(String[] args) {
      // input - don''t change this code
      Scanner scn = new Scanner(System.in);
      int marks = scn.nextInt();
      // input - don''t change this code
      if(marks>90)
      System.out.println("excellent");
      if(marks>80&&marks<=90)
      System.out.println("good");
    if(marks>70&&marks<=80)
      System.out.println("fair");
    if(marks>60&&marks<=70)
      System.out.println("meets expectation");
      }
  }